swap_meet/vendor.py

```diff
@@ -0,0 +1,100 @@
+from swap_meet.item import Item
+
+# ---------- Wave 1 -----------
+
+class Vendor:
+    def __init__(self, inventory = None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory    
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item 
+    
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+    
+    
+    
+    
+# --------------- Wave 3 --------------   
+    def swap_items(self,friend_vendor,my_item,their_item):  
```

Comment:
This all works great! That said, this method would be a good place to make use of the add and remove methods above